src/keyfile.c

```diff
@@ -86,12 +86,10 @@
 #define GEANY_DEFAULT_TOOLS_BROWSER		"open -a safari"
 #define GEANY_DEFAULT_FONT_SYMBOL_LIST	"Helvetica Medium 12"
 #define GEANY_DEFAULT_FONT_MSG_WINDOW	"Menlo Medium 12"
-#define GEANY_DEFAULT_FONT_EDITOR		"Menlo Medium 12"
```

Comment:
IIUC the overrides are for distros to use, and Geany OSX is effectively a distro run by @techee so it makes sense.